compile.bat

```diff
@@ -1,11 +1,11 @@
-set HOMEDRIVE=C:
-set HOMEPATH=\
-d:
-cd \php\xdebug
-d:\cvs upd -dP
-
-nmake /f xdebug_4_3.mak "CFG=xdebug - Win32 Release_TS" clean all
-copy Release_TS\php_xdebug.dll c:\xdebug-4.3-2.0dev.dll
-
-nmake /f xdebug.mak "CFG=xdebug - Win32 Release_TS" clean all
-copy Release_TS\php_xdebug.dll c:\xdebug-5.0-2.0dev.dll
+set HOMEDRIVE=C:
+set HOMEPATH=\
+d:
+cd \php\xdebug
+d:\cvs upd -dP
+
+nmake /f xdebug_4_3.mak "CFG=xdebug - Win32 Release_TS" clean all
+copy Release_TS\php_xdebug.dll c:\xdebug-4.3-2.0dev.dll
+
+nmake /f xdebug.mak "CFG=xdebug - Win32 Release_TS" clean all
+copy Release_TS\php_xdebug.dll c:\xdebug-5.0-2.0dev.dll

```

Comment:
This doesn't look like a related change. In fact, this file should not even be in the repository (it still uses `cvs`!!!). I'll make sure to remove it separately.

Reply:
bat MUST be formatted with `\r\n` - if it should be removed completely, fine, no functional change